docs/runbooks/NoLeadingVirtOperator.md

```diff
@@ -1,42 +1,73 @@
 
-## Impact
+The `virt-operator` is the first operator to start in a cluster. Its primary responsibilities include the following: 
+
+- Installing, live updating, and live upgrading a cluster
+
+- Monitoring the life cycle of top-level controllers, such as `virt-controller`, `virt-handler`, `virt-launcher`, and managing their reconciliation
+
+- Certain cluster-wide tasks, such as certificate rotation and infrastructure management
 
-The virt-operator is the first k8s operator coming alive in a KubeVirt cluster. Its primary responsibilities are installation, live-update, live-upgrade of a KubeVirt cluster, monitoring the life-cycle of top-level controllers, such as virt-controller, virt-handler, virt-launcher, etc. and manage their reconciliation. In addition, virt-operator is responsible for cluster-wide tasks, such as certificate rotation and some infrastructure management, etc.  The virt-operator deployment has a default replica of 2 pods, with one leader pod holding a leader lease, indicating an operating virt-operator pod. 
+The `virt-operator` deployment has a default replica of 2 pods, with one leader pod holding a leader lease, indicating an operating `virt-operator` pod. 
 
-This alert indicates a failure at the level of the KubeVirt cluster. Critical cluster-wide management functionalities such as certification rotation, KubeVirt upgrade, and reconciliation of KubeVirt controllers, may not be available for the time being.
+## Impact
+
+This alert indicates a failure at the level of the cluster. As a result, critical cluster-wide management functionalities, such as certification rotation, upgrade, and reconciliation of controllers, might be currently not available.
 
 ## Diagnosis
 
-The information on the leader status of a virt-operator pod can be deduced from the pod logs. The log messages containing `Started leading` and `acquire leader` should help deduce the leader status of a given virt-operator pod. 
 
-In addition, always check whether there are any running virt-operator pods and their status, with following commands:
-- `export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"`
-- `kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-operator`
-- `kubectl -n $NAMESPACE logs <pod-name>`.
-- `kubectl -n $NAMESPACE describe pod <pod-name>`.
+1. Configure the `NAMESPACE` environment variable:
+```bash
+$ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+```
 
+2. Obtain the details of the `virt-operator` pods:
+```bash
+$ kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-operator
+```
 
-***Leader pod example:***
+3. Inspect `virt-operator` pod logs:
+```bash
+$ kubectl -n $NAMESPACE logs <virt-operator>
 ```
+
+Log messages that contain `Started leading` or `acquire leader` indicate the leader status of a given `virt-operator` pod.
+
+***Leader pod example:***
+```bash
 $ kubectl -n $NAMESPACE logs <pod-name> |grep lead
```

Comment:
Remove the command from this code block. We just need the output.

Also, remove "bash" from ```bash, since it is no longer a bash command. (if you leave 'bash', the preview colors the first word or character of each line when the file is converted to Asciidoc. looks very strange)